File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F09.P2.ts
const xs: number[] = [3, 1, 2]
const a: number = 1
const b: number = 2
// BEGIN PROBE F09.P2
xs.sort((p, q) => q - p)
const lt = a < b
// END PROBE F09.P2
console.log(xs.join(","), lt)

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F09.P2.java
import java.util.ArrayList;
import java.util.Comparator;
import java.util.List;

class Main {
    static String probe() {
        var xs = new ArrayList<>(List.of(2, 3, 1));
        int a = 4;
        int b = 9;
        // BEGIN PROBE F09.P2
        xs.sort(Comparator.reverseOrder());
        boolean lt = a < b;
        // END PROBE F09.P2
        return xs + " " + lt;
    }

    public static void main(String[] args) {
        System.out.println(probe());
    }
}
